generators/angular/generator.mts

```diff
@@ -164,6 +164,23 @@ export default class AngularGenerator extends BaseApplicationGenerator<Generator
     return this.delegateTasksToBlueprint(() => this.postWritingEntities);
   }
 
+  get end() {
+    return this.asEndTaskGroup({
+      end({ application }) {
+        this.log.ok('Angular application generated successfully.');
```

Comment:
this.logger.log?

Reply:
Looks ok to me:
![6380DB39-DE55-4EFB-A1C8-C6005E10ECAE](https://user-images.githubusercontent.com/1417183/221357298-07ffda28-77f3-43f2-8d88-805c6ae33c97.jpeg)

Reply:
It is, but it won't centralize log management for this entry